.github/workflows/linters.yml

```diff
@@ -18,20 +18,20 @@ jobs:
         run: npm install -g @lhci/cli@0.11.x
       - name: Lighthouse Report
         run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: "18.x"
-      - name: Setup Webhint
-        run: |
-          npm install --save-dev hint@7.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
-      - name: Webhint Report
-        run: npx hint .
+  # webhint:
+  #   name: Webhint
+  #   runs-on: ubuntu-22.04
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: actions/setup-node@v3
+  #       with:
+  #         node-version: "18.x"
+  #     - name: Setup Webhint
+  #       run: |
+  #         npm install --save-dev hint@7.x
+  #         [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+  #     - name: Webhint Report
+  #       run: npx hint .
   stylelint:
     name: Stylelint
     runs-on: ubuntu-22.04
```

Comment:
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but please make sure the check for `eslint` is running properly this means there should be 5 checks in total (counting `webhint`). I noticed you forgot to add some lines of code for checking `eslint` in your `linters.yml` file above, I've attached below the code you need for that so please copy it just above line `49`. Please visit this [repo](https://github.com/microverseinc/linters-config/blob/master/html-css-js) for more info.
```yml
eslint:
    name: ESLint
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: "18.x"
      - name: Setup ESLint
        run: |
          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
      - name: ESLint Report
        run: npx eslint .
```